docs/src/rules/no-extra-parens.md

```diff
@@ -83,8 +83,6 @@ typeof (a);
 
 (Object.prototype.toString.call());
 
-(function(){} ? a() : b());
-
```

Comment:
This example is not flagged by the rule, I'm not sure why it was there.

Reply:
It seems that when it was added the rule was indeed reporting it (https://github.com/eslint/eslint/issues/3527), which was wrong. It's best to just remove this example 👍